scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -783,21 +783,210 @@ async def start_reborn_server(
+            except Exception as trace_exc:
+                diagnostics.record(
+                    "diagnostic_error",
+                    source="trace_stop",
+                    error=repr(trace_exc),
+                )
             raise
         finally:
+            try:
+                diagnostics.write_summary()
+            except Exception:
+                pass
```

Ruff (0.15.20)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)